sql/2023/01/cwvs-by-wordpress-version.sql

```diff
@@ -0,0 +1,106 @@
+# HTTP Archive query to get Core Web Vitals "good" rates by WordPress version.
+#
+# WPP Research, Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See https://github.com/GoogleChromeLabs/wpp-research/pull/26
+
+CREATE TEMP FUNCTION IS_GOOD(good FLOAT64, needs_improvement FLOAT64, poor FLOAT64)
+RETURNS BOOL
+AS (
+  good / (good + needs_improvement + poor) >= 0.75
+);
+
+CREATE TEMP FUNCTION IS_NON_ZERO(good FLOAT64, needs_improvement FLOAT64, poor FLOAT64)
+RETURNS BOOL
+AS (
+  good + needs_improvement + poor > 0
+);
+
+SELECT
```

Comment:
Similar to my feedback in https://github.com/GoogleChromeLabs/wpp-research/pull/17#discussion_r1065247612, can you please format the query?

Note that in some instances the BigQuery formatter will format in a way that is worse than here, specifically in the case when using slightly more complex formulas. I would suggest the following:
* Start with formatting the query in the editor.
* Add a blank line each between the overall query and the separate `CREATE TEMP FUNCTION` statements.
* Manually reformat "ugly to read" broken-apart lines to put them back in one line. Here those are specifically:
    * The `COUNT(DISTINCT IF(...` statements.
    * The `IS_NON_ZERO(...` and `IS_GOOD(...` statements.

TLDR: The BigQuery formatter does a good job for the most part, particularly to ensure correct indentation, but it's not great when using slightly more complex formulas so we can adjust those lines by hand to be easy to read.